nvflare/apis/shareable.py

```diff
@@ -12,8 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import pickle
-
+from ..fuel.utils import fobs
```

Comment:
minor stuff: we might want to use absolute import for fobs

Reply:
Good idea. Relative and wildcard imports should be disallowed.